wallet-new/src/Cardano/Wallet/Server/Plugins.hs

```diff
@@ -204,23 +206,23 @@ walletBackend protocolMagic (NewWalletBackendParams WalletBackendParams{..}) (pa
         env <- ask
         let diffusion' = Kernel.fromDiffusion (lower env) diffusion
         bracketKernelActiveWallet protocolMagic passiveLayer passiveWallet diffusion' $ \active _ -> do
-          ctx <- view shutdownContext
-          let
-            portCallback :: Word16 -> IO ()
-            portCallback port = usingLoggerName "NodeIPC" $ flip runReaderT ctx $ startNodeJsIPC port
-          walletServeImpl
-            (getApplication active)
-            walletAddress
-            -- Disable TLS if in debug modeit .
-            (if isDebugMode walletRunMode then Nothing else walletTLSParams)
-            Nothing
-            (Just portCallback)
+            ctx <- view shutdownContext
+            let
+                portCallback :: Word16 -> IO ()
+                portCallback port = usingLoggerName "NodeIPC" $ flip runReaderT ctx $ startNodeJsIPC port
+            walletServeImpl
+                (getApplication active)
+                walletAddress
+                -- Disable TLS if in debug modeit .
+                (if isDebugMode walletRunMode then Nothing else walletTLSParams)
+                Nothing
+                (Just portCallback)
```

Comment:
spacing change to bring code into compliance with 4 space indentation

Reply:
Side-note but, better make all those "linting" changes or import re-organizing in a separated commit with a clear message. It helps _A LOT_ during rebases / merge-conflicts :)

Reply:
hmm! I tend to do squash-n-merge on GitHub when I am done with a PR, which obviously rmeoves that distinction. Should I just do merge commits instead?